passman.py

```diff
@@ -77,7 +78,18 @@ def btn_Edit():
         renderCreateView(index=ind)
 
     def btn_CopyPassword():
-        print "copying password"
+        current_selection = entries_list.curselection() 
+        if current_selection:
+            selected_entry = entries_list.get(current_selection[0])
```

Comment:
I totally agree. That was wrong approach. Actually it came in my mind today morning that I submitted wrong code. Anyways thanks.